container/docker/handler.go

```diff
@@ -168,7 +168,8 @@ func newDockerContainerHandler(
 		ignoreMetrics:      ignoreMetrics,
 	}
 
-	if !ignoreMetrics.Has(container.DiskUsageMetrics) {
+	if !ignoreMetrics.Has(container.DiskUsageMetrics) && storageDriver != devicemapperStorageDriver {
```

Comment:
Is this necessary? `otherStorageDir` directory is common to all storage drivers and we do want to surface usage from that.